### ISSUE TITLE
Update the conflux recipe

### ISSUE THREAD

**Issue by queicherius** [gw2efficiency/issues#1454]
There should be no more slumbering recipe instead 2 conflux recipe (and same for pvp amulet)
2 conflux recipes 1 is identical to old slumbering one while other is old recipe. Old recipe can only be done by ppl that had a slumbering before this patch. Maybe can add that as a warning to those 2 recipes specifically.

---

remove the old recipe, change the old slumbering into real one for both and should be no confusion
basically hide the old big recipe since vendor is cheaper

**Comment by queicherius** [comment]
Live.
